sympy/functions/elementary/tests/test_trigonometric.py

```diff
@@ -857,6 +857,8 @@ def test_atan_rewrite():
     assert atan(x).rewrite(asec) == sqrt(x**2)*asec(sqrt(x**2 + 1))/x
     assert atan(x).rewrite(acsc) == (-acsc(sqrt(x**2 + 1)) + pi/2)*sqrt(x**2)/x
 
+    assert atan(5*I).evalf() == atan(x).rewrite(log).evalf(subs={x:5*I})
```

Comment:
Thanks, this should take care of the cut from `I` to `oo*I`. To fully cover both pieces of the cut another point on the negative imaginary axis is needed. Similarly, the branch cut of `acot` consists of two pieces separated by the singularity at the origin.